test/source/tests/gmail.ts

```diff
@@ -301,6 +302,7 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
       await createSecureDraft(t, browser, gmailPage, 'reply draft');
       await createSecureDraft(t, browser, gmailPage, 'offline reply draft', { offline: true });
       await gmailPage.page.reload({ waitUntil: 'networkidle2' });
+      await Util.sleep(30);
```

Comment:
That's quite long, but I supposed you tried shorter and it didn't work?

Reply:
In unstable connections, it didn't work.
And had to use 30 seconds for unstable internet